### ISSUE TITLE
Application wizard: dirty project should not be selectable as existing project (validate before starting to merge openshift app into existing proj and fail)

### ISSUE THREAD

**Issue by zhouhuaian** [zhouhuaian/redhat_bot_test#144]
EXEC: Import a git based project into the workspace in JBT (ex. git@github.com:adietish/kitchensink.git)
EXEC: Launch New application wizard from OpenShift explorer (or launch via project context menu Configure->New/Import OpenShift Application).
EXEC: Choose basic cartridge and click Next button (ex. EAP 6).
EXEC: Fill in name on the next wizard page and click Next.
EXEC: Choose Use existing project and select the existing git based project.
EXEC: Finish the wizard.

*RESULT:* There is the error that there are uncommitted changes and it is required to commit them first. Application on OpenShift is default (from basic cartridge).
!import-failed-uncommitted-changes.png!
This is expected as you cannot merge into a project with uncommitted changes. But the missing requirement (clean project without uncommitted changes) can be spotted far earlier when the existing project is selected.
